www/docusaurus.typedoc.js

```diff
@@ -0,0 +1,23 @@
+function generateTypedocDocusaurusPlugins(directories) {
+  return directories.map((directory, idx) => [
+    'docusaurus-plugin-typedoc',
+    {
+      id: directory,
+      entryPoints: [`../packages/${directory}/src/index.ts`],
+      tsconfig: `../packages/${directory}/tsconfig.build.json`,
+      out: `./typedoc/${directory}`,
```

Comment:
this output must be added to the turbo pipeline

Reply:
this is probably why there is only react and client package in the deployed version:

<img width="344" alt="CleanShot 2023-02-28 at 11 17 56@2x" src="https://user-images.githubusercontent.com/51714798/221825134-cfff2cb1-f2e3-48d7-a785-bb70921a81d3.png">

turbo didn't see any changes to server or next so probably skipped it or something?

Reply:
Do you know what commands exactly are run during deployment? Is is possible to reproduce the same build env locally?

Reply:
same command. if you remove the build & typedoc folders and rerun build-www, you should hit the turbo cache and it'll restore the build output but not typedoc (since you haven't told it that it outputs that)

Reply:
Run in root to build with turbo:

```sh
turbo --filter www build
```

Reply:
That's quite odd, as I verified everything just now, and it appears that turbo caches contain all necessary files.

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/1523305/221831811-565a47d1-1407-4331-a30c-4539861e6c73.png">

Let's see if adding this generated content to the pipeline will do the job.

Reply:
It did not 👀 Could someone maybe check the Vercel logs whether `Documentation generated at ./docs/typedoc` is correctly executed for all 4 packages?

Reply:
it's not in the build output

<img width="319" alt="CleanShot 2023-02-28 at 11 58 21@2x" src="https://user-images.githubusercontent.com/51714798/221834505-3330754b-c89d-4e02-80e7-a52e72dc8d06.png">

Reply:
Strange... 
![CleanShot 2023-02-28 at 12 00 33@2x](https://user-images.githubusercontent.com/51714798/221835001-bb4b205e-f2b3-4af3-b8e0-c98777666581.png)

Reply:
Ah, they are not generated due to build errors, as typedoc will skip those.

Reply:
Rebased on top of `main`, as there were some changes to `pnpm-lock` in the meantime. Lets hope it helps.